index.js

```diff
@@ -638,19 +628,28 @@ class LatticeKeyring extends EventEmitter {
         return resolve();
       }
       try {
+        // Stup an SDK client
```

Comment:
```suggestion
        // Setup an SDK client
```

😅

Reply:
😓

Reply:
Last commit was an excellent candidate to squash in some comment changes [4b98b1f](https://github.com/GridPlus/eth-lattice-keyring/pull/29/commits/4b98b1f1bbe4a7ab0eadc6c89bcf3bafbf5abe81)